tests/all/stack_overflow.rs

```diff
@@ -2,6 +2,7 @@ use std::sync::atomic::{AtomicUsize, Ordering::SeqCst};
 use wasmtime::*;
 
 #[test]
+#[cfg_attr(feature = "experimental_x64", ignore)] // TODO #1079 require probe stacks
```

Comment:
This mentions 1079 but should it be 2079?

Reply:
Good catch, thanks